utils/tx.ts

```diff
@@ -989,13 +989,13 @@ export async function registerAsset(
 
 export async function registerL1Asset(
   sudoUser: User,
-  assetId: BN,
-  locMarker: BN,
+  assetId: any,
```

Comment:
you can use extended type instead, like 
assetId: BN | null ,
I think that we should keep types there instead of using "any"

Reply:
Done